### ISSUE TITLE
[Android][Bug]Incorrect marker icon cache

### ISSUE THREAD

**Issue by cloudhx** [nativescript-community/ui-mapbox#80]
Check out below code:
https://github.com/nativescript-community/ui-mapbox/blob/0f4df978a606dfd02bbcc17f2c60081d270d2abf/src/ui-mapbox/index.android.ts#L1348

When setting different icons for markers, all markers will only show icon from the first marker as cache key is using `marker.iconPath` which is always undefined.

Currently the workaround is to also set `iconPath` along with `icon`, e.g
```typescript
const markers = [
      {
        id: "3fbad6ba-e3b8-4c01-bfc1-d7487a148b0a",
        lat: 55.6815768675545,
        lng: 12.590348950181465,
        icon: "res://icon1",
        iconPath: "icon1",
      },
      {
        id: "e45be1ca-0398-43e0-8a17-3b4ed4f338e2",
        lat: 55.681975196092644,
        lng: 12.592566894399852,
        icon: "res://icon2",
        iconPath: "icon2",
      },
    ];
```

**Comment by seros00** [comment]
Hi, @cloudhx.

There is a PR to solve this issue:
[Update index.android.ts - Fixes #32 Same icon for all markers added](https://github.com/nativescript-community/ui-mapbox/pull/77)

**Comment by cloudhx** [comment]
Thanks @seros00, I will hereby close this issue.
